docs/intro/tutorial01.txt

```diff
@@ -430,7 +430,7 @@ look like this:
         'django.contrib.messages',
         'django.contrib.staticfiles',
         'polls',
```

Comment:
There's nothing wrong with trailing commas; I think in multi line lists trailing commas are much preferable. The problem with tuples is the requirement of a trailing comma in a length-one tuple (which is likely not multi line).